src/fileformat_pb.jl

```diff
@@ -0,0 +1,104 @@
+# Automatically generated by ProtoBuf.jl
+# syntax: proto2
+using ProtoBuf
+import ProtoBuf.meta
+
+mutable struct Blob <: ProtoType
+    __protobuf_jl_internal_meta::ProtoMeta
+    __protobuf_jl_internal_values::Dict{Symbol,Any}
+    __protobuf_jl_internal_defaultset::Set{Symbol}
+
+    function Blob(; kwargs...)
+        obj = new(meta(Blob), Dict{Symbol,Any}(), Set{Symbol}())
+        values = obj.__protobuf_jl_internal_values
+        symdict = obj.__protobuf_jl_internal_meta.symdict
+        for nv in kwargs
+            fldname, fldval = nv
+            fldtype = symdict[fldname].jtyp
+            (fldname in keys(symdict)) || error(string(typeof(obj), " has no field with name ", fldname))
+            if fldval !== nothing
+                values[fldname] = isa(fldval, fldtype) ? fldval : convert(fldtype, fldval)
+            end
+        end
+        obj
+    end
+end # mutable struct Blob
+const __meta_Blob = Ref{ProtoMeta}()
+function meta(::Type{Blob})
+    ProtoBuf.metalock() do
+        if !isassigned(__meta_Blob)
+            __meta_Blob[] = target = ProtoMeta(Blob)
+            fnum = Int[2,1,3,4,5,6,7]
+            allflds = Pair{Symbol,Union{Type,String}}[:raw_size => Int32, :raw => Vector{UInt8}, :zlib_data => Vector{UInt8}, :lzma_data => Vector{UInt8}, :OBSOLETE_bzip2_data => Vector{UInt8}, :lz4_data => Vector{UInt8}, :zstd_data => Vector{UInt8}]
```

Comment:
Again I do not use `ProtoBuf`, but id we control it  `Union{Type,Symbol}` would look better to me.

Reply:
You means `Pair{Symbol,Type}` ?